baby-gru/src/components/card/MoorhenCarbohydrateCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { moorhen } from "../../types/moorhen";
+import { useSelector } from 'react-redux';
+import {Card, Col, Row } from "react-bootstrap";
+import {useCallback, useEffect, useRef, useState} from "react";
+import { guid } from "../../utils/MoorhenUtils";
+import {PrivateerResultsEntry} from "../../types/privateer";
+
+export const MoorhenCarbohydrateCard = (props: {
+    carbohydrate: PrivateerResultsEntry;
+    molecule: moorhen.Molecule;
+}) => {
+
+    const anchorEl = useRef(null)
+    const width = useSelector((state: moorhen.State) => state.sceneSettings.width)
+
+    const { carbohydrate, molecule } = props
+    const molecules = useSelector((state: any) => state.molecules);
+
+    const handleClick = useCallback(async (e) => {
+        const newCenterString =
+            e.target.dataset.chainid +
+            '/' +
+            e.target.dataset.seqnum +
+            '(' +
+            e.target.dataset.resname +
+            ')';
+
+        if (molecule !== null) {
+            await molecule.centreOn(newCenterString);
+        }
+    }, [molecules]);
```

Comment:
Looking at the code I think this hook does not depend on `molecules`, it should instead have an empty dependency array.